for_003_nrt_rw1_glad_deforestation_alerts/contents/src/__init__.py

```diff
@@ -0,0 +1,521 @@
+import os
+import sys
+import datetime
+import logging
+import requests
+import ee
+import re
+import time
+import json
+
+# name of image collection in GEE where we will upload the final data
+EE_COLLECTION = 'projects/resource-watch-gee/for_003_nrt_rw1_glad_deforestation_alerts'
+
+# name of the image collection in GEE where the original data is stored  
+EE_COLLECTION_ORI = 'projects/glad/alert/UpdResult'
+
+# do you want to delete everything currently in the GEE collection when you run this script?
+CLEAR_COLLECTION_FIRST = False
+
+# how many assets can be stored in the GEE collection before the oldest ones are deleted?
+MAX_ASSETS = 1
```

Comment:
It is always good to have at least two assets. If you only have 1, at the end, the old one will be deleted and you will only have the new asset. If for some reason you find an error with the script today, you have no functioning assets that you could go back to while you fix whatever is wrong.
```suggestion
MAX_ASSETS = 2
```